docs/data-sources/netflow_monitor_policy.md

```diff
@@ -0,0 +1,77 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Tenant Policies"
+layout: "aci"
+page_title: "ACI: aci_netflow_monitor_policy"
+sidebar_current: "docs-aci-data-source-aci_netflow_monitor_policy"
+description: |-
+  Data source for Netflow Monitor Policy
+---
+
+# aci_netflow_monitor_policy #
+
+Data source for Netflow Monitor Policy
+
+## API Information ##
+
+* Class: [netflowMonitorPol](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/netflowMonitorPol/overview)
+
+
+* Distinguished Name Formats:
+  - `uni/infra/monitorpol-{name}`
+  - `uni/tn-{name}/monitorpol-{name}`
+
+## GUI Information ##
+
+* Location: `Tenants -> Policies -> NetFlow -> NetFlow Monitors`
```

Comment:
Should we add both locations for the DN format possibilities??

Reply:
Yes, you're right I forgot to add it. I will do it.